lib/model/yake_keywords.py

```diff
@@ -26,15 +43,29 @@ def run_yake(self, text: str,
         :param num_of_keywords: int
         :returns: str
         """
+        ### if language is set to "auto", auto-detect it.
+        if language == 'auto':
+            language = cld3.get_language(text).language
+        ### replace special characters
+        replacement = {"`": "'",
+                       "‘": "'",
+                       "“": "\""}
```

Comment:
We're still missing one of the smart double quotes here:
> We need to replace `“` and `”` .

Reply:
I also think we may have the wrong character in the first entry? We're missing the single closing smart quote: `’`